Library/Homebrew/cask/lib/hbc/exceptions.rb

```diff
@@ -149,4 +149,23 @@ def to_s
       EOS
     end
   end
+
+  class CaskQuarantineError < CaskError
+    attr_reader :path, :reason
+
+    def initialize(path, reason)
+      @path = path
+      @reason = reason
+    end
+
+    def to_s
+      "Failed to quarantine #{path}" << (reason.empty? ? "." : ": #{reason}")
```

Comment:
Interpolate this instead.

Reply:
I've copied these from `CaskInvalidError` (earlier in the file). Would you like me to change all of them to interpolation or just the quarantine ones?

Reply:
Done and done 😆